modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -58,7 +58,7 @@ NOTE: Before upgrading to ownCloud Server 10.12, you MUST upgrade to PHP 7.4. ow
 
 === Persistent major file versions workflow
 
-Restore operation logic changed. Now restore creates a new current version of the file from one of the past noncurrent versions of the file. The current version also receives incremented mtime for the file, and author of the file is the user who restored it. The old noncurrent version is no longer removed upon restore and the current version no longer receives mtime of that version. The current version of the file is now shown in the Versions Tab, highlighted with "gray" background. Versions now persist additional extended metadata on versioning tags, that allow easier identification of the versions. Each update increases the minor version for the file. The current version of the file now can be published, which increases the major version tag. Each new edit of the file creates noncurrent versions - the ones tagged with major version due to publishing will persist long term and won't be subject to any retention policies. Migrate from deprecated save_version_author to save_version_metadata. https://github.com/owncloud/core/pull/40531[#40531]
+To save space, old versions of files are ususally deleted over time according to ownCloud's retention policy. Users are now able to mark the current version as "published". This increases the major version tag and prevents deletion. Also, restore operation logic changed. Now restore creates a new current version of the file from one of the past noncurrent versions of the file. The current version also receives incremented mtime for the file, and author of the file is the user who restored it. The old noncurrent version is no longer removed upon restore and the current version no longer receives mtime of that version. The current version of the file is now shown in the Versions Tab, highlighted with "gray" background. Versions now persist additional extended metadata on versioning tags, that allow easier identification of the versions. Each update increases the minor version for the file. Each new edit of the file creates noncurrent versions - the ones tagged with major version due to publishing will persist long term and won't be subject to any retention policies. Migrate from deprecated save_version_author to save_version_metadata. https://github.com/owncloud/core/pull/40531[#40531]
```

Comment:
`usually`

Reply:
I am overall unhappy with the term "publish" in this context. That would imply broader visibility to me. That is not the case at all. I'd be scared to click on a publish link, when it is a sensitive document. But that is what it was named in the UI.